src/templates/_macros/app_tile.html

```diff
@@ -12,6 +12,7 @@
       src -- if tile is a link, attach a src param for analytics purposes
   #}
   {% set app = apps.transform(app) %}
+  {% set name = app.name if is_detail else app.short_name %}
```

Comment:
We want to use `short_name` for websites on the listing view but `name` on the detail view. I think this is required somewhere.